### PR TITLE
Fix timeout in WebServer::_uploadReadByte and handleClient() (#9990)

### PR DESCRIPTION
**Pull request by TD-er**
Fixes: #9990

-----------
## Description of Change
See #9990 

In short:
- WebServer::_uploadReadByte() does not keep track of some timeout
- WebServer::handleClient() does only set timeout to client after the function call where it is needed.

### CODE REVIEW

**Review by P-R-O-C-H-Y — APPROVED** [review]
LGTM

File: libraries/WebServer/src/WebServer.h [hunk 1]
```diff
@@ -88,7 +88,7 @@ typedef struct {
   HTTPRawStatus status;
   size_t totalSize;    // content size
   size_t currentSize;  // size of data currently in buf
-  uint8_t buf[HTTP_UPLOAD_BUFLEN];
+  uint8_t buf[HTTP_RAW_BUFLEN];
```
Comment (me-no-dev) [inline libraries/WebServer/src/WebServer.h:91]: why is this change needed?

Comment (TD-er) [reply libraries/WebServer/src/WebServer.h:91]: Because somewhere else this buffer is used with this defined size :)
Was a copy/paste error.

**Review by me-no-dev — APPROVED** [review]
